client/blocks/comment-button/index.jsx

```diff
@@ -18,29 +18,46 @@ import { getPostTotalCommentsCount } from 'state/comments/selectors';
 
 class CommentButton extends Component {
 	static propTypes = {
-		onClick: PropTypes.func,
-		tagName: PropTypes.string,
 		commentCount: PropTypes.number,
+		link: PropTypes.string,
+		onClick: PropTypes.func,
 		showLabel: PropTypes.bool,
+		tagName: PropTypes.string,
+		target: PropTypes.string,
 	};
 
 	static defaultProps = {
-		onClick: noop,
-		tagName: 'li',
-		size: 24,
 		commentCount: 0,
+		link: null,
+		onClick: noop,
 		showLabel: true,
+		size: 24,
+		tagName: 'li',
+		target: null,
 	};
 
 	render() {
-		const { translate, commentCount, onClick, showLabel, tagName: containerTag } = this.props;
+		const {
+			commentCount,
+			link,
+			onClick,
+			showLabel,
+			tagName: containerTag,
```

Comment:
I know this is not part of this PR, but is there a reason why we're renaming `tagName` into `containerTag`?
Is it to improve the `React.createElement` readability? Feels equally readable to me.

(It's not an attempt to be consistent with React docs because [they use `type`](https://reactjs.org/docs/react-api.html#createelement)).

Reply:
No idea; removed it.